app/Http/Controllers/Areas/AreasController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers\Areas;
+
+use App\Models\DBC\Area\AreasAndZones as AreasAndZonesWotlk;
+use App\Models\WOD\DBC\Area\AreasAndZones as AreasAndZonesWod;
+use App\Exceptions\UnsupportedVersion;
+use App\Helpers\TCAPI;
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use Illuminate\Routing\Controller;
+
+class AreasController extends Controller
+{
+    private $api;
+
+    /**
+     * AchievementsController constructor.
+     *
+     * @param TCAPI $api
+     */
+    public function __construct(TCAPI $api)
```

Comment:
You could make TCAPI to a singleton. So you don't need to initialize it for each controller